cms/static/cms/sass/components/_structureboard.scss

```diff
@@ -8,19 +8,8 @@
     z-index: z(structure, base);
     width: 100%;
     height: 100%;
+    background: $structure-bgcolor;
 
-    .cms-structure-dimmer {
```

Comment:
yeah I know, just my thoughts on making it general if we might need it somewhere else (currently only used within the sideframe after this removal)